condition/types.go

```diff
@@ -102,6 +102,9 @@ type Condition struct {
 	// Version identifies the revision number for this struct.
 	Version string `json:"version"`
 
+	// Client is the user/jwt user that requested the condition.
```

Comment:
Why do we need this data here?

Reply:
We discussed this and this is required if we want to keep track of who requested the Condition.